TestAssets/TestProjects/DependencyContextFromTool/project.json

```diff
@@ -4,15 +4,16 @@
   },
   "dependencies": {},
   "frameworks": {
-    "netstandardapp1.5": {
+    "netcoreapp1.0": {
       "imports": [
+        "netstandardapp1.5",
```

Comment:
fairly certain I added this back in explicitly but I can't remember, I'll take it out and we'll see if it fails again